src/include/cell.i

```diff
@@ -60,17 +69,17 @@ __cell_check_value_validity(WT_SESSION_IMPL *session, wt_timestamp_t durable_sta
  */
 static inline void
 __cell_pack_value_validity(WT_SESSION_IMPL *session, uint8_t **pp, wt_timestamp_t durable_start_ts,
-  wt_timestamp_t durable_stop_ts, wt_timestamp_t start_ts, uint64_t start_txn,
+  wt_timestamp_t start_ts, uint64_t start_txn, wt_timestamp_t durable_stop_ts,
   wt_timestamp_t stop_ts, uint64_t stop_txn, bool prepare)
 {
     uint8_t flags, *flagsp;
 
     __cell_check_value_validity(
-      session, durable_start_ts, durable_stop_ts, start_ts, start_txn, stop_ts, stop_txn);
+      session, durable_start_ts, start_ts, start_txn, durable_stop_ts, stop_ts, stop_txn);
 
     /* Globally visible values have no associated validity window, set a flag bit and store them. */
-    if (start_ts == WT_TS_NONE && start_txn == WT_TXN_NONE && stop_ts == WT_TS_MAX &&
-      stop_txn == WT_TXN_MAX)
+    if (durable_start_ts == WT_TS_NONE && start_ts == WT_TS_NONE && start_txn == WT_TXN_NONE &&
+      durable_stop_ts == WT_TS_NONE && stop_ts == WT_TS_MAX && stop_txn == WT_TXN_MAX)
```

Comment:
Why isn't `durable_stop_ts` checked against `WT_TS_MAX`?

There are a bunch of places where this happens (either testing or initialization as a default), so I'm sure there's a reason.

Reply:
we want to follow the same initial value for all the durable timestamps across the code as both of them will store the maximum value of the start/stop timestamps. Refer WT-5668 for more details.